### PR TITLE
Adding game class.

### PR DESCRIPTION
**Pull request by Rurek404**

### CODE REVIEW

File: game_class.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+class Game:
+    def __init__(self, board_size, who_starts = 'x'):
+        self.board_size = board_size
+        self.board = self._create_board()
+        self.whose_turn_is_it = who_starts
+    
+    def _create_board(self):
+        list1 = []
```
Comment (jkochel) [inline game_class.py]: I don't like this name, it's not very clear what it is for, could you change it?

File: game_class.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        board = []
+        for i in range(self.board_size):
```
Comment (jkochel) [inline game_class.py]: can you change i to something more verbose?

File: game_class.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    
+    def _create_header(self):
+        s2 = "    "
+        for i in range(self.board_size):
+            s2 += str(i + 1) + "   "
+        
+        return s2
+        
+    
+
```
Comment (jkochel) [inline game_class.py]: too many blank lines

**Review by jkochel — CHANGES_REQUESTED** [review]

File: game_class.py (hunk 1 above)
Comment (Rurek404) [reply game_class.py]: Updated

File: game_class.py (hunk 2 above)
Comment (Rurek404) [reply game_class.py]: Updated

File: game_class.py (hunk 3 above)
Comment (Rurek404) [reply game_class.py]: Done

**Review by jkochel — APPROVED** [review]

**Pull request merged**